docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+
+services:
+  web:
+    container_name: apiserver
+    restart: always
+    build: .
+    ports:
+      - '5000:3000'
+    links:
```

Comment:
You don't need to add this tag to reach the other service. By default, services can be reached with their names. If you want to state there is a dependency between services, you can use `depends_on`

https://docs.docker.com/compose/compose-file/#links